swan-lake/learn/using-the-cli-tools.md

```diff
@@ -155,41 +155,39 @@ These everyday commands are your best friends! They address the very basics of p
 </tr>
 </table>
 
-## Module Commands
+## Package Commands
 
-These commands allow you to work with the Ballerina Central to share Ballerina modules with others in a safe, secure, and dependable way.
+Ballerina packages are the way to organize real-world Ballerina development tasks. Some of them allow you to work with the Ballerina Central also to share Ballerina packages with others in a safe, secure, and dependable way.
```

Comment:
```suggestion
Ballerina packages are the way to organize real-world Ballerina development tasks. The last 3 commands given below allow you to work with the Ballerina Central and also to share Ballerina packages with others in a safe, secure, and dependable way.
```

Reply:
We do not need to mention the specifics in the intro.

Reply:
+1

Reply:
Fixed.